boards/arm/mimx8mp_phyboard_pollux/mimx8mp_phyboard_pollux_defconfig

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2022 PHYTEC Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SOC_SERIES_IMX8ML_M7=y
+CONFIG_SOC_MIMX8ML8=y
+CONFIG_BOARD_MIMX8MP_PHYBOARD_POLLUX=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_CONSOLE=y
+CONFIG_XIP=y
```

Comment:
If `CONFIG_XIP=y` here, then why do you provide defaults for `FLASH_BASE_ADDRESS` and `FLASH_SIZE` when it is disabled in the `Kconfig.defconfig` for this board?